01 OperatorOverloading/OperatorOverloading.Host/Host.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading;
+
+namespace OperatorOverloading
+{
+    class Host
+    {
+        static void Main(string[] args)
+        {
+           
+            double amount;
+            string currency;
+            Money moneyOne;
+            Money moneyTwo;
+            Money moneyThree;
+
+            try
+            {
+
+
+                Console.WriteLine("Enter the Curreny type  USD/INR and the amount ");
+                currency = Console.ReadLine();
+                double.TryParse(Console.ReadLine(), out amount);
```

Comment:
That's why a constructor which accepts `100 USD` helps the consumer. By doing so all the validation is done by the `Money` class.